govcd/vdc.go

```diff
@@ -1038,7 +1038,10 @@ func (vdc *Vdc) CreateStandaloneVmAsync(params *types.CreateVmParams) (Task, err
 	}
 	params.XmlnsOvf = types.XMLNamespaceOVF
 
-	return vdc.client.ExecuteTaskRequest(href, http.MethodPost, types.MimeCreateVmParams, "error creating standalone VM: %s", params)
+	// 37.1 Introduced new parameters to VM configuration
+	return vdc.client.ExecuteTaskRequestWithApiVersion(href, http.MethodPost,
+		types.MimeCreateVmParams, "error creating standalone VM: %s", params,
+		vdc.client.GetSpecificApiVersionOnCondition(">=37.1", "37.1"))
```

Comment:
Do I understand correctly that we risk to remove `EFI` settings if we perform an API call on API v37.0 for any of the POST/PUT calls for VM? Area all these endpoints verified in SDK to avoid breaking/removing EFI settings

Reply:
This shouldn't be the case for endpoints unrelated to BootOptions or Firmware, but I'll run some more manual/automatic tests to check